src/components/calcite-select/calcite-select.e2e.ts

```diff
@@ -62,12 +62,23 @@ describe("calcite-select", () => {
       await internalSelect.asElement().select("dos");
       await page.waitForChanges();
 
-      const selected = await page.findAll("calcite-option[selected]");
+      let selected = await page.findAll("calcite-option[selected]");
 
       await assertSelectedOption(page, selected[0]);
       expect(selected.length).toBe(1);
       expect(selected[0].innerText).toBe("dos");
       expect(spy).toHaveReceivedEventTimes(1);
+
+      const [, , lastOption] = await page.findAll("calcite-option");
```

Comment:
nitpick: the `, , ` is slightly confusing. Maybe `await page.findAll("calcite-option")[2];` would be better? or `await page.findAll("calcite-option:last-child");`?